recipes/ratatosk/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -x
+
+git clone https://github.com/pmelsted/bifrost.git
```

Comment:
Can't you use the bifrost conda package instead? https://anaconda.org/bioconda/bifrost

Reply:
With the current code, I am not sure because bifrost is built at the same time as ratatosk: https://github.com/DecodeGenetics/Ratatosk/blob/b08613bc947a3df628b2c2f018414e0e6d6f8593/src/CMakeLists.txt#L7

So this CMakeLists file won't be valid anymore I think ?

But I can suggest the Ratatosk developers to remove bifrost from their code and add it as a requirement.

Reply:
Maybe it would work by just adding bifrost as a requirement + skipping this git clone step, can you try? ("sur un malentendu ça peut passer")

Reply:
arf, it fails... can you ask the Ratatosk developers? if they have a quick solution it would be good, otherwise we'll merge and see later I guess

Reply:
Ratatosk requires a different `MAX_KMER_SIZE` compiling option (64) that the one used to compile the existing conda package for bifrost (default: 32): https://github.com/DecodeGenetics/Ratatosk/issues/45#issuecomment-1311435804
I guess we can keep the cloning and build step for bifrost ?

Reply:
ping :-)

Reply:
ok yep it makes sense, can you revert the last commit, and then I'll merge